From 057080a6c824149f5b510ae001dfd03724073043 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Thu, 10 Apr 2025 08:56:04 +0200
Subject: [PATCH] Support UnifiedNlp (microG)

  from

https: //github.com/syberia-project/platform_frameworks_base/commit/6ce5ffc2279d0fa5a615e75711b716361cdad644
Change-Id: I8fd05e376c83fd3bc9a7fb4b58242851a00f330d
---
 core/res/res/values/config.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index 8ab160060d7f..11d6cb8ba254 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -1660,6 +1660,8 @@
         <item>com.android.location.fused</item>
         <!-- Google Play Services or microG (free reimplementation) location provider -->
         <item>com.google.android.gms</item>
+        <!-- microG unifiednlp location provider (standalone) -->
+        <item>org.microg.nlp</item>
     </string-array>
 
     <!-- This string array can be overriden to enable test location providers initially. -->
-- 
2.25.1

